administrator/language/de-DE/com_scheduler.ini

```diff
@@ -147,6 +166,8 @@ COM_SCHEDULER_TEST_RUN_STATUS_TERMINATED="Status: Beendet"
 COM_SCHEDULER_TEST_RUN_TASK="Aufgabe: „%s“"
 COM_SCHEDULER_TEST_RUN_TITLE="Aufgabe (ID: %d)"
 COM_SCHEDULER_TEST_TASK="Aufgabe"
+COM_SCHEDULER_TOOLBAR_PURGE="Verlauf löschen"
+COM_SCHEDULER_TOOLBAR_PURGE_CONFIRM="Sicher, dass der gesamte Ausführungsverlauf gelöscht werden soll?"
```

Comment:
```suggestion
COM_SCHEDULER_TOOLBAR_PURGE_CONFIRM="Sind Sie sicher, dass der gesamte Ausführungsverlauf gelöscht werden soll?"
```

Reply:
Wollte "Sie" vermeiden.

Reply:
@zero-24 verwenden wir sonst auch so in anderen Fällen.

Reply:
Fine for me.